packages/tenant-consumer/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "pagopa-interop-tenant-event-consumer",
+  "private": true,
+  "version": "1.0.0",
+  "description": "PagoPA Interoperability tenant consumer service that updates the read model when events are stored",
+  "main": "dist",
+  "type": "module",
+  "scripts": {
+    "lint": "eslint . --ext .ts,.tsx",
+    "lint:autofix": "eslint . --ext .ts,.tsx --fix",
+    "format:check": "prettier --check src",
+    "format:write": "prettier --write src",
+    "start": "node --watch --no-warnings --loader ts-node/esm -r 'dotenv-flow/config' ./src/index.ts",
+    "build": "tsc"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@pagopa/eslint-config": "^3.0.0",
+    "@types/dotenv-flow": "^3.2.0",
+    "@types/node": "^20.3.1",
+    "prettier": "^2.8.8",
+    "ts-node": "^10.9.1",
+    "typescript": "^5.1.3"
+  },
+  "dependencies": {
+    "@jm18457/kafkajs-msk-iam-authentication-mechanism": "^3.1.2",
```

Comment:
looking [this code](https://github.com/pagopa/interop-be-monorepo/pull/100/files#diff-3203c3253260bee2355742709e598781328ef0b2a02f91c7c7199192d25b35b7R3) it's a leftover to remove, IAM authentication now will be handled by [middleware](https://github.com/pagopa/interop-be-monorepo/pull/103)